src/layers.js

```diff
@@ -1,105 +1,30 @@
 const get = require('lodash.get')
 
 exports.layerInfo = {
-    java: getJavaLayerProps,
+    java: {
+        layerName: 'thundra-lambda-java-layer',
+        thundraHandlerName:
+            'io.thundra.agent.lambda.core.handler.ThundraLambdaHandler',
+        needHandlerDelegation: true,
+    },
     dotnetcore: {
         layerName: 'thundra-lambda-dotnetcore21-layer',
-        defaultLayerVersion: '6',
         needHandlerDelegation: false,
         customRuntime: true,
     },
     python: {
         layerName: 'thundra-lambda-python-layer',
-        defaultLayerVersion: '38',
         thundraHandlerName: 'thundra.handler.wrapper',
         needHandlerDelegation: true,
     },
-    node: getNodeLayerProps,
-    layerAwsAccountNo: 269863060030,
-    delegatedHandlerEnvVarName: 'thundra_agent_lambda_handler',
-}
-
-function getNodeLayerProps(func, service, userLayerVersion) {
-    const optsWithCR = {
+    node: {
```

Comment:
I think, we can simply use `thundra_handler.wrapper` as Thundra handler instead of long `/opt/nodejs/node_modules/@thundra/core/dist/handler.wrapper`